test/erlumts_nas_security_tests.erl

```diff
@@ -50,26 +52,104 @@ ts33401_test_sets() ->
0f8a637116a27ad04b444a0932dd60fbd12671c11e1c0ec73e789879faa3d42c64d20cd1252742a3768c25a901585888ecee1e612d9936b403b0775949a66cdfd99a29b1345baa8d9d5400c91024b0a607363b013ce5de9ae869d3b8d95b0570b3c2d391422d32450cbcfae96652286e96dec1214a9346527980a8192eac1c39a3aaf6f15351da6be764df89772ec0407d06e4415befae7c92580df9bf507497c8f2995160d4e218daacb02944abf83340ce8be1686a960faf90e2d90c55cc6475babc3171a80a363174954955d7101dab16ae8179167e21444b443a9eaaa7c91de36d118c39d389f8dd4469a846c9a262bf7fa18487a79e8de11699e0b8fdf557cb48719d453ba713056109b93a218c89675ac195fb4fb06639b3797144955b3c9327d1aec003d42ecd0ea98abf19ffb4af3561a67e77c35bf15c59c2412da881db02b1bfbcebfac5152bc99bc3f1d15f771001b7029fedb028f8b852bc4407eb83f891c9ca733254fdd1e9edb56919ce9fea21c174072521c18319a54b5d4efbebddf1d8b69b1cbf25f489fcc981372547cf41d008ef0bca1926f934b735e090b3b251eb33a36f82ed9b29cf4cb944188fa0e1e38dd778f7d1c9d987b28d132dfb9731fa4f4b416935be49de30516af3578581f2f13f561c0663361941eab249a4bc123f8d15cd711a956a1bf20fe6eb78aea2373361da0426c79a530c3bb1de0c99722ef1fde39ac2b00a0a8ee7c800a08bc2264f89f4effe627ac2f0531fb554f6d21d74c590a70adfaa390bdfbb3d68e46215cab187d2368d5a71f5ebec081cd3b20c082dbe4cd2faca28773795d6b0c10204b659a939ef29bbe1088243624429927a7eb576dd3a00ea5e01af5d47583b2272c0c161a806521a16ff9b0a722c0cf26b025d5836e2258a4f7d4773ac801e4263bc294f43def7fa8703f3a4197463525887652b0b2a4a2a7cf87f00914871e25039113c7e1618da34064b57a43c463249fb8d05e0f26f4a6d84972e7a9054824145f91295cdbe39a6f920facc659712b46a54ba295bbe6a90154e91b33985a2bcd420ad5c67ec9ad8eb7ac6864db272a516bc94c2839b0a8169a6bf58e1a0c2ada8c883b7bf497a49171268ed15ddd2969384e7ff4bf4aab2ec9ecc6529cf629e2df0f08a77a65afa12aa9b505df8b287ef6cc91493d1caa39076e28ef1ea028f5118de61ae02bb6aefc3343a050292f199f401857b2bead5e6ee2a1f191022f9278016f047791a9d18da7d2a6d27f2e0e51c2f6ea30e8ac49a0604f4c13542e85b68381b9fdcfa0ce4b2d341354852d360245c536b612af71f3e77c9095ae2dbde504b265733dabfe10a20fc7d6d32c21ccc72b8b3444ae663d65922d17f82caa2b865cd88913d291a65899026ea1328439723c198c36b0c3c8d085bfaf8a320fde334b4a4919b44c2b95f6e8ecf73393f7f0d2a40e60b1d406526b022ddc331810b1a5f7c347bd53ed1f105d6a0d30aba477e178889ab2ec55d558deab2630204336962b4db5b663b6902b89e85b31bc6af50fc50accb3fb9b57b663297031378db47896d7fbaf6c600add2c67f936db037986db856eb49cf2db3f7da6d23650e438f1884041b013119e4c2ae5af37cccdfb68660738b58b3c59d1c0248437472aba1f35ca1fb90cd714aa9f635534f49e7c5bba81c2b6b36fdee21ca27e347f793d2ce944edb23c8c9b914be10335e350feb5070394b7a4a15c0ca120283568b7bfc254fe838b137a2147ce7c113a3a4d65499d9e86b87dbcc7f03bbd3a3ab1aa243ece5ba9bcf25f82836cfe473b2d83e7a7201cd0b96a72451e863f6c3ba664a6d073d1f7b5ed990865d978bd3815d06094fc9a2aba5221c22d5ab996389e3721e3af5f05beddc2875e0dfaeb39021ee27a41187cbb45ef40c3e73bc03989f9a30d12c54ba7d2141da8a875493e65776ef35f97debc2286cc4af9b4623eee902f840c52f1b8ad658939aef71f3f72b9ec1de21588bd35484ea44436343ff95ead6ab1d8afb1b2a303df1b71e53c4aea6b2e3e9372be0d1bc99798b0ce3cc10d2a596d565dba82f88ce4cff3b33d5d24e9c0831124bf1ad54b792532983dd6c3a8b7d0">>),
```

Comment:
that's a long binary... possible to break it up like this, or could we hide it in a macro-file?
```suggestion
        hexstream_to_binary(<<"00000000000000000101010101010101"
                              "e0958045f3a0bba4e3968346f0a3b8a7"
                              "c02a018ae640765226b987c913e6cbf0"
                              "83570016cf83efbc61c082513e21561a"
                              "427c009d28c298eface78ed6d56c2d45"
                              "05ad032e9c04dc60e73a81696da665c6"
                              "c48603a57b45ab33221585e68ee31691"
                              "87fb0239528632dd656c807ea3248b7b"
                              "46d002b2b5c7458eb85b9ce95879e034"
                              "0859055e3b0abbc3eace8719caa80265"
                              "c97205d5dc4bcc902fe1839629ed7132"
                              "8a0f0449f588557e6898860e042aecd8"
                              "4b2404c212c9222da5bf8a89ef679787"
                              "0cf50771a60f66a2ee62853657addf04"
                              "cdde07fa414e11f12b4d81b9b4e8ac53"
                              "8ea30666688d881f6c348421992f31b9"
                              "4f8806ed8fccff4c9123b89642527ad6"
                              "13b109bf75167485f1268bf884b4cd23"
                              "d29a0934925703d634098f7767f1be74"
                              "91e708a8bb949a3873708aef4a36239e"
                              "50cc08235cd5ed6bbe578668a17b58c1"
                              "171d0b90e813a9e4f58a89d719b11042"
                              "d6360b1b0f52deb730a58d58faf46315"
                              "954b0a872691475977dc88c0d733feff"
                              "54600a0cc1d0300aaaeb94572c6e95b0"
                              "1ae90de04f1dce47f87e8fa7bebf77e1"
                              "dbc20d6ba85cb9143d518b285dfa04b6"
                              "98bf0cf7819f20fa7a288eb0703d995c"
                              "59940c7c66de57a9b70f82379b70e203"
                              "1e450fcfd2181326fcd28d8823baaa80"
                              "df6e0f443559647539fd8907c0ffd9d7"
                              "9c130ed81c9afd9b7e848c9fed38443d"
                              "5d380e53fbdb8ac8c3d3f06876054f12"
                              "2461107de92fea09c6f6923a188d53af"
                              "e54a10f60e6e9d5a03d996b5fbc820f8"
                              "a637116a27ad04b444a0932dd60fbd12"
                              "671c11e1c0ec73e789879faa3d42c64d"
                              "20cd1252742a3768c25a901585888ece"
                              "e1e612d9936b403b0775949a66cdfd99"
                              "a29b1345baa8d9d5400c91024b0a6073"
                              "63b013ce5de9ae869d3b8d95b0570b3c"
                              "2d391422d32450cbcfae96652286e96d"
                              "ec1214a9346527980a8192eac1c39a3a"
                              "af6f15351da6be764df89772ec0407d0"
                              "6e4415befae7c92580df9bf507497c8f"
                              "2995160d4e218daacb02944abf83340c"
                              "e8be1686a960faf90e2d90c55cc6475b"
                              "abc3171a80a363174954955d7101dab1"
                              "6ae8179167e21444b443a9eaaa7c91de"
                              "36d118c39d389f8dd4469a846c9a262b"
                              "f7fa18487a79e8de11699e0b8fdf557c"
                              "b48719d453ba713056109b93a218c896"
                              "75ac195fb4fb06639b3797144955b3c9"
                              "327d1aec003d42ecd0ea98abf19ffb4a"
                              "f3561a67e77c35bf15c59c2412da881d"
                              "b02b1bfbcebfac5152bc99bc3f1d15f7"
                              "71001b7029fedb028f8b852bc4407eb8"
                              "3f891c9ca733254fdd1e9edb56919ce9"
                              "fea21c174072521c18319a54b5d4efbe"
                              "bddf1d8b69b1cbf25f489fcc98137254"
                              "7cf41d008ef0bca1926f934b735e090b"
                              "3b251eb33a36f82ed9b29cf4cb944188"
                              "fa0e1e38dd778f7d1c9d987b28d132df"
                              "b9731fa4f4b416935be49de30516af35"
                              "78581f2f13f561c0663361941eab249a"
                              "4bc123f8d15cd711a956a1bf20fe6eb7"
                              "8aea2373361da0426c79a530c3bb1de0"
                              "c99722ef1fde39ac2b00a0a8ee7c800a"
                              "08bc2264f89f4effe627ac2f0531fb55"
                              "4f6d21d74c590a70adfaa390bdfbb3d6"
                              "8e46215cab187d2368d5a71f5ebec081"
                              "cd3b20c082dbe4cd2faca28773795d6b"
                              "0c10204b659a939ef29bbe1088243624"
                              "429927a7eb576dd3a00ea5e01af5d475"
                              "83b2272c0c161a806521a16ff9b0a722"
                              "c0cf26b025d5836e2258a4f7d4773ac8"
                              "01e4263bc294f43def7fa8703f3a4197"
                              "463525887652b0b2a4a2a7cf87f00914"
                              "871e25039113c7e1618da34064b57a43"
                              "c463249fb8d05e0f26f4a6d84972e7a9"
                              "054824145f91295cdbe39a6f920facc6"
                              "59712b46a54ba295bbe6a90154e91b33"
                              "985a2bcd420ad5c67ec9ad8eb7ac6864"
                              "db272a516bc94c2839b0a8169a6bf58e"
                              "1a0c2ada8c883b7bf497a49171268ed1"
                              "5ddd2969384e7ff4bf4aab2ec9ecc652"
                              "9cf629e2df0f08a77a65afa12aa9b505"
                              "df8b287ef6cc91493d1caa39076e28ef"
                              "1ea028f5118de61ae02bb6aefc3343a0"
                              "50292f199f401857b2bead5e6ee2a1f1"
                              "91022f9278016f047791a9d18da7d2a6"
                              "d27f2e0e51c2f6ea30e8ac49a0604f4c"
                              "13542e85b68381b9fdcfa0ce4b2d3413"
                              "54852d360245c536b612af71f3e77c90"
                              "95ae2dbde504b265733dabfe10a20fc7"
                              "d6d32c21ccc72b8b3444ae663d65922d"
                              "17f82caa2b865cd88913d291a6589902"
                              "6ea1328439723c198c36b0c3c8d085bf"
                              "af8a320fde334b4a4919b44c2b95f6e8"
                              "ecf73393f7f0d2a40e60b1d406526b02"
                              "2ddc331810b1a5f7c347bd53ed1f105d"
                              "6a0d30aba477e178889ab2ec55d558de"
                              "ab2630204336962b4db5b663b6902b89"
                              "e85b31bc6af50fc50accb3fb9b57b663"
                              "297031378db47896d7fbaf6c600add2c"
                              "67f936db037986db856eb49cf2db3f7d"
                              "a6d23650e438f1884041b013119e4c2a"
                              "e5af37cccdfb68660738b58b3c59d1c0"
                              "248437472aba1f35ca1fb90cd714aa9f"
                              "635534f49e7c5bba81c2b6b36fdee21c"
                              "a27e347f793d2ce944edb23c8c9b914b"
                              "e10335e350feb5070394b7a4a15c0ca1"
                              "20283568b7bfc254fe838b137a2147ce"
                              "7c113a3a4d65499d9e86b87dbcc7f03b"
                              "bd3a3ab1aa243ece5ba9bcf25f82836c"
                              "fe473b2d83e7a7201cd0b96a72451e86"
                              "3f6c3ba664a6d073d1f7b5ed990865d9"
                              "78bd3815d06094fc9a2aba5221c22d5a"
                              "b996389e3721e3af5f05beddc2875e0d"
                              "faeb39021ee27a41187cbb45ef40c3e7"
                              "3bc03989f9a30d12c54ba7d2141da8a8"
                              "75493e65776ef35f97debc2286cc4af9"
                              "b4623eee902f840c52f1b8ad658939ae"
                              "f71f3f72b9ec1de21588bd35484ea444"
                              "36343ff95ead6ab1d8afb1b2a303df1b"
                              "71e53c4aea6b2e3e9372be0d1bc99798"
                              "b0ce3cc10d2a596d565dba82f88ce4cf"
                              "f3b33d5d24e9c0831124bf1ad54b7925"
                              "32983dd6c3a8b7d0">>),
```

Reply:
Broke up all the long lines. Hmm I don't see why this is possible... :exploding_head: 

```
12> <<"12345678" "12345678">>.
<<"1234567812345678">>
13>
13> <<"12345678""12345678">>.
<<"1234567812345678">>
14>
14> <<"12345678" , "12345678">>.
<<"1234567812345678">>
```

Reply:
https://www.erlang.org/docs/24/reference_manual/data_types.html#string

it's an Erlang magic trick :)

I realized this week that you can do it with maps as well
```
1> #{key1 => a} #{key2 => b}.
#{key1 => a,key2 => b}
```
which might make things confusing if they are in a list and you forget the comma separator.....